### PR TITLE
Added EmailEngine

### PR DESCRIPTION
**Pull request by andris9**
EmailEngine allows accessing email accounts using a REST API instead of IMAP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,6 +196,7 @@ The definition of developer-first for this repo is:
 * [ngrok](https://ngrok.com/) - Generate public URLs for internal servers (behind NAT/firewall).
 * [zigi](https://www.zigi.ai/) - Developer’s assistant for mundane non-coding tasks via Slack.
 * [Nylas](https://www.nylas.com/) - APIs for productivity workflows (email, calendar, contacts...) - like plaid for productivity.
+* [EmailEngine](https://emailengine.app/) - API and webhooks for existing email accounts
```
Comment (agamm) [inline README.md:199]:
```suggestion
* [EmailEngine](https://emailengine.app/) - API and webhooks for existing email accounts (IMAP & SMTP).
```

**Review by agamm — CHANGES_REQUESTED** [review]
Thanks for the contribution, looks like a great tool. 
It is quite the oddball here in the SaaS space :) but it still complies with the contribution guide, so I'll accept it.
Also, I would move this to the `Mail` category.
I'll change the description to make it work.

File: README.md (hunk 1 above)
Comment (agamm) [reply README.md:199]: @andris9 ?

Comment (andris9) [reply README.md:199]: Looks great 👍

**Comment by agamm** [comment]
@andris9 Can you move it to the mail category?

**Comment by agamm** [comment]
@andris9 ?

**Comment by agamm** [comment]
@andris9 shall I close or are we changing it?

**Comment by agamm** [comment]
Closing until there is a response 👀

**Pull request closed without merging**